### PR TITLE
Lh/processing fee

### PR DESCRIPTION
**Pull request by laholstege**

### CODE REVIEW

File: Anlab.Mvc/Client/components/specs/Summary.spec.tsx [hunk 1]
```diff
@@ -21,7 +21,8 @@ describe('<Summary />', () => {
                                                 hideError={true}
                                                 isCreate={true}
                                                 onSubmit={null}
-                                                testItems={testItems} />);
```
Comment (laholstege) [inline Anlab.Mvc/Client/components/specs/Summary.spec.tsx]: @jSylvestre I wasn't sure how to properly adjust the spec for this so I just added processingFee as null for this. Is that okay?

File: Anlab.Mvc/Controllers/OrderController.cs [hunk 2]
```diff
@@ -39,8 +42,13 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
         public async Task<IActionResult> Create()
         {
             var joined = await  _orderService.PopulateTestItemModel();
+            var proc = await _labworksService.GetPrice("PROC");
```
Comment (srkirkland) [inline Anlab.Mvc/Controllers/OrderController.cs]: @laholstege move this into a const or an app setting

File: Anlab.Core/Models/OrderDetails.cs [hunk 3]
```diff
@@ -29,6 +29,8 @@ public class OrderDetails
             }
         }
         public string ClientId { get; set; }
+        public decimal InternalProcessingFee { get; set; }
+        public decimal ExternalProcessingFee { get; set; }
```
Comment (srkirkland) [inline Anlab.Core/Models/OrderDetails.cs]: @laholstege I'd rather you dump this and just have one processingFee property.  It'll take a small amount of extra work up front, but just a little bit.  That part in react where you say `const processingFee = this.state.payment.clientType === 'uc' ? this.state.internalProcessingFee : this.state.externalProcessingFee; ` you'll have to change to do the math.Ceil(this.state.processingFee) logic, and of course you'll have to pass the appsettings.NonUcRate into the react view in order to do that.  But then every other place in here where you have two ProcessingFee props can go away

File: Anlab.Mvc/Client/components/specs/Summary.spec.tsx (hunk 1 above)
Comment (jSylvestre) [reply Anlab.Mvc/Client/components/specs/Summary.spec.tsx]: @laholstege Yes, that is fine. That is what I would have done.

File: Anlab.Mvc/Controllers/OrderController.cs (hunk 2 above)
Comment (jSylvestre) [reply Anlab.Mvc/Controllers/OrderController.cs]: I'd say Const.

**Pull request merged**